core/src/calleeFunctions/index.ts

```diff
@@ -95,6 +167,12 @@ export const getMarketDataRecords = async function (
     return marketDataRecords;
 };
 
+export const getMarketDataRecords = memoizee(_getMarketDataRecords, {
+    promise: true,
+    maxAge: 30 * 1000,
+    length: 4,
+});
```

Comment:
Why was this change necessary? Anyway, `amount` here is an object, so it might not get cached properly (will be cached forever or not at all depending on how the amount is created)

Reply:
So do you want to remove it?

Reply:
yes, i confused the names 😢